keyboards/xenon/keymaps/default/config.h

```diff
@@ -0,0 +1,25 @@
+/* Copyright 2020 Kyrre Havik Eriksen
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#ifdef OLED_DRIVER_ENABLE
+  #define OLED_DISPLAY_128X32
+#endif
```

Comment:
This should probably defined at the keyboard level. 

And if nothing else is in this file, it can be removed. 

Also, it should be: 
```c
#ifdef OLED_ENABLE
#    define OLED_DISPLAY_128X32
#endif
```